### ISSUE TITLE
ColorVariable incorrectly warns on colors in lists

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#936]
If I were to define a list, like a shadow:

```scss
$my-shadow: 2px 2px 2px rgba(0, 0, 0, 0.2);
$my-other-shadow: 2px 2px 2px #f00;
```

I currently get a ColorVariable warning for each line. ☹️
